awscli/customizations/cloudtrail/verifyqueryresults.py

```diff
@@ -64,17 +61,14 @@ def validate(self, public_key_base64, sign_file):
         :param sign_file: Dict of sign file data returned when JSON
             decoding a manifest.
         """
-        try:
-            public_key = self._load_public_key(public_key_base64)
-            signature_bytes = binascii.unhexlify(sign_file["hashSignature"])
-            hashing = hashes.SHA256()
-            public_key.verify(
-                signature_bytes,
-                self._create_string_to_sign(sign_file),
-                PKCS1v15(),
-                hashing,
-            )
-        except cryptography.exceptions.InvalidSignature:
+        public_key = self._load_public_key(public_key_base64)
+        signature_bytes = binascii.unhexlify(sign_file["hashSignature"])
+        result = public_key.verify(
+            signature_algorithm=RSASignatureAlgorithm.PKCS1_5_SHA256,
+            digest=hashlib.sha256(self._create_string_to_sign(sign_file)).digest(),
+            signature=signature_bytes
+        )
+        if not result:
```

Comment:
Should add same comment from validate module about the previously raised cryptography error.

```suggestion
        if not result:
        # The previous implementation caught a cryptography.exceptions.InvalidSignature
        # exception here and re-raised the DigestSignatureError to avoid the stack trace
        # leaking any information about the key.
```